elide-async/src/main/java/com/yahoo/elide/async/operation/TableExportOperation.java

```diff
@@ -110,9 +111,10 @@ public AsyncAPIResult call() {
      * @param prevScope RequestScope object.
      * @param projection Entity projection.
      * @return Observable PersistentResource
+     * @throws URISyntaxException URISyntaxException from malformed or incorrect URI
```

Comment:
For what URI will this exception be thrown?

Reply:
for parsing Queryparam & path from querystring from getRequestScope() in JSONAPITableExportOperation

Reply:
Does it make sense for every TableExportOperation to parse URIs?  If not, we shouldn't generalize the exception handling here in this way.